plugins/available/battery.plugin.bash

```diff
@@ -2,15 +2,23 @@ cite about-plugin
 about-plugin 'display info about your battery charge level'
 
 ac_adapter_connected(){
-    if command_exists acpi;
-    then
-        acpi -a | grep "on-line"
-        if [[ "$?" -eq 0 ]]; then
-           return 1
-        else
-           return 0
-        fi
```

Comment:
You're right, this changes the behavior to simplify things returning the grep exit code, so the "if" is not needed to "swap" the 0 and the 1.

In the OSX case, I supposed by looking at the code that the behavior is 0 when connected, and 1 when not, Am I wrong?